Proper Learn Intro Page (#4917)
w3f/polkadot-wiki: docs/learn/learn-index.md

---
id: learn-index

> title: Learn Polkadot

Reviewer: ```suggestion
title: Learn about Polkadot
```

sidebar_label: Learn Polkadot

> description: Learn how the Polkadot network works, from the ground up.

Reviewer: ```suggestion
description: Learn how the Polkadot network works and what tool you can use to interact with it.
```

keywords: [learn, index, tools, resources]
slug: ../learn-index
---

Welcome to the learner's section of the Polkadot Wiki.

> Here, you will be able to learn how the Polkadot network works from the ground up - from how
> accounts on Polkadot, to the protocols that power the Polkadot network. If you are looking to begin
> your learning journey, whether you're a developer or someone who is just getting started with
> blockchain.

Reviewer: ```suggestion
Whether you are a developer or someone just getting started with
blockchain, here you will learn how the Polkadot network works from the ground up - from accounts to the protocols that power the Polkadot network.
```

> There are a series of video tutorials which are also provided, which you may
> [find here](./learn-video-tutorials.md).

Reviewer: ```suggestion
If you want to learn more about how to use different tools on Polkadot, you can visit the [Polkadot Technical Explainers playlist](https://youtube.com/playlist?list=PLOyWqupZ-WGuAuS00rK-pebTMAOxW41W8) on YouTube.
```

## The Basics: DOT, Accounts, & Staking

> Learn the basics of transactions ([extrinsics](./learn-extrinsics)), accounts, staking
> [here](./learn-accounts).

Reviewer: ```suggestion
Learn the basics of transactions, accounts, staking, NFTs, and more.
```

- [Accounts](./learn-accounts.md) - How accounts work on Polkadot.
- [Transactions (extrinsics)](./learn-extrinsics.md) - The role of extrinsics, and how they define
  state changes on Polkadot.
- [Tokens and Assets](./learn-assets.md) - See how different types of assets can exist in the
  Polkadot ecosystem and how asset teleportation works.
- [NFTs](./learn-nft.md) - Polkadot's approach to NFTs.
- [Architectural Components](./learn-polkadot-host) - Learn the basics of how the Polkadot
  architecture works.
- [Staking](./learn-staking.md) - Learn about how staking works, and how it aids in securing
  Polkadot.

## Advanced Concepts

> Learn more advanced concepts of the Polkadot network, as well as numerous
> [how-to guides for staking](./learn-staking-guides):

Reviewer: ```suggestion
Learn more advanced concepts of the Polkadot network.
```

- [Accounts - Advanced](./learn-account-advanced) - Learn how accounts work on a finer, more
  advanced level.
- [Advanced Staking Concepts](./learn-staking-advanced.md) - Advanced staking information.
- [NFT Pallets](./learn-nft-pallets.md) - An overview of the functionalities of NFT Pallets in
  Polkadot Ecosystem.
- [Cryptography](./learn-cryptography.md) - Understand what cryptography is used throughout the
  Polkadot network.
- [NPoS Algorithms](./learn-phragmen) - Learn about the election method used on Polkadot's Nominated
  Proof of Staking.

## Network Architecture

> Learn more about Polkadot's network architecture in depth:

Reviewer: ```suggestion
Learn more about Polkadot's network architecture in depth.
```

- [Architecture](./learn-architecture.md) - Learn about the key components to Polkadot's
  Architecture.
- [Nominator](./learn-nominator.md) - Learn about what it means to be a nominator.
- [Validators](./learn-validator.md) - An introduction about validators.
- [Collators](./learn-collator.md) - Learn about how collators maintain the network.
- [Parachains](./learn-parachains.md) - An introductory guide to parachains.
- [Bridges](./learn-bridges.md) - Learn about bridges and common bridge examples.
- [Cross-Consensus Messaging (XCM)](./learn-xcm.md) - Learn about the messaging format at the
  forefront of interoperability.
- [SPREE (Shared Protected Runtime Execution Enclaves)](./learn-spree.md) - Learn about the
  fundamentals of SPREE.

## Polkadot Comparisons

> Learn how Polkadot compares to other similar technologies:

Reviewer: ```suggestion
Learn how Polkadot compares to other similar technologies.
```

- [Kusama](./learn-comparisons-kusama.md) - How the canary network, Kusama, differs to Polkadot.
- [Ethereum](./learn-comparison-ethereum-2.md) - A high-level comparison between Polkadot and
  Ethereum.
- [Cosmos](./learn-comparisons-cosmos.md) - Learn about the differences between Polkadot and Cosmos.
- [Avalanche](./learn-comparisons-avalanche.md) - A high-level comparison between Polkadot and
  Avalanche.
- [Other comparisons](./learn-comparisons.md) - Learn how Polkadot compares to Ethereum, Cosmos, and
  other chains.
- [Layer Two and Rollups](./learn-comparison-rollups.md) - Comparisons between various rollup and
  layer two protocols and how they relate to Polkadot.

> ## Future Implementations & Upcoming Features

Reviewer: ```suggestion
## Future Implementations
```

> Learn upcoming innovations for the Polkadot network that are not yet released:

Reviewer: ```suggestion
Learn about upcoming innovations for the Polkadot network that are not yet implemented.
```
